SecretSantaServer/SecretSantaServer/appsettings.Development.json

```
@@ -4,5 +4,13 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "Jwt": {
+    "Issuer": "secret-santa",
+    "Audience": "",
+    "Key": "k3V7s9p2xQrTnM1ZbL8fGhYvWq4XeJ5NcR0aUoP6iDc="
```

для дев мода лучше явно указать поддельность токена 
```suggestion
    "Key": "super-secret-jwt-key"
```

---

нет.